src/core/error.js

```diff
@@ -67,10 +66,10 @@ export function createErrorVargs(var_args) {
 /**
  * Rethrows the error without terminating the current context. This preserves
  * whether the original error designation is a user error or a dev error.
- * @param {...*} var_args
+ * @param {...*} args
  */
-export function rethrowAsync(var_args) {
-  const error = createErrorVargs.apply(null, arguments);
+export function rethrowAsync(...args) {
+  const error = createErrorVargs(...args);
```

Comment:
Doing `...args` to collect args into an array, then `...args` to spread it, is considerably slower than just `...arguments`. Prefer `arguments` whenever possible.

Reply:
Reverted;I mistakenly assumed that the implementation would be more favorable to such a standard use of the feature 🤦🏻